core-bundle/assets/controllers/image-size-controller.js

```diff
@@ -29,6 +41,30 @@ export default class extends Controller {
         }
     }
 
+    _updateInputs() {
+        const widthInput = this.inputTargets[0];
+        const heightInput = this.inputTargets[1];
```

Comment:
shouldnt these be two actual Stimulus targets?

Reply:
I would think `withTarget` and `heightTarget`, yes. Because we can safely assume both must be there, and there will never be any other than these 2.